xenoclassify.wdl

```diff
@@ -2,60 +2,100 @@ version 1.0
 
 # imports workflows for the top portion of WGSPipeline
 import "imports/bwaMem.wdl" as bwaMem
+import "imports/pull_star.wdl" as star
 
 workflow xenoClassify {
 input {
         File fastqR1
-	File? fastqR2
-	String refHost  = "$MM10_BWA_INDEX_ROOT/mm10.fa"
-	String refGraft = "$HG19_BWA_INDEX_ROOT/hg19_random.fa"
+	File fastqR2
+	String refHost    = "$MM10_BWA_INDEX_ROOT/mm10.fa"
+	String refGraft   = "$HG19_BWA_INDEX_ROOT/hg19_random.fa"
+        String libraryDesign = "WG"
         String rG = "'@RG\\tID:TEST-RUN_XENO\\tLB:XENOTEST\\tPL:ILLUMINA\\tPU:TEST-RUN_XENO\\tSM:TEST_XENOTEST_X'"
-        String bwaMemModules = "bwa/0.7.17 samtools/1.9 hg19-bwa-index/0.7.17 mm10-bwa-index/0.7.17"
+        String alignerModules = "bwa/0.7.17 samtools/1.9 hg19-bwa-index/0.7.17 mm10-bwa-index/0.7.17"
```

Comment:
I wonder should we add modules for star here as well?

Reply:
as we discussed, this parameter is going to be set in the olive. Production workflow will not rely on the default value, it will be always set differently depending on libraryDesign